lib/govuk_publishing_components/version.rb

```diff
@@ -1,3 +1,3 @@
 module GovukPublishingComponents
-  VERSION = "21.38.4".freeze
+  VERSION = "21.39.4".freeze
```

Comment:
@oscarwyatt we don't modify the version number of the gem as part of a code change PR. Changes are made as normal PRs and then when a new version of the gem is needed we follow [this procedure](https://github.com/alphagov/govuk_publishing_components/blob/master/docs/publishing-to-rubygems.md).

Reply:
e.g. https://github.com/alphagov/govuk_publishing_components/pull/1429